api-docs/generator.py

```diff
@@ -242,8 +242,9 @@ def output_markdown(sections, scenarios, section_mapping):
             payload = dict(
                 title=title,
                 sidebar_order=i,
-                description=''.join(endpoint['text']),
+                description=' '.join(endpoint['text']),
```

Comment:
I'm not sure how this is generated, but should this be a newline instead? I discovered the issue because one of the descriptions which is supposed to have multiple paragraphs was rendered as a single line post-markdown.

Reply:
This would be a good test case: https://docs.sentry.io/api/events/get-project-group-index/?platform=javascript

Reply:
This data is a list of all the lines of text in the doc string. You are right that joining on space will drop paragraph breaks. I'll switch to newline.